tests/class-schedule.e2e.ts

```diff
@@ -0,0 +1,36 @@
+import LoginPage from './pageobjects/login.po';
+import NavbarPage from './pageobjects/navbar.po';
+import { testEnvironment } from './config/config';
+import ClassesPage from './pageobjects/classes.po';
+
+const loginPage = new LoginPage();
+const navbar = new NavbarPage();
+const classesPage = new ClassesPage();
+
+fixture(`Class Schedule tests`)
+  .page(testEnvironment.feUrl)
+  .beforeEach(async (t) => {
+    await loginPage.loginAsPrinciple();
+    await navbar.navigateToClassesPage();
+    await classesPage.createNewScheduleTestClass();
+  });
+
+test('should open popup on click on cell', async (t) => {
+  await t.click(classesPage.scheduleCell);
+  await t.expect(classesPage.editCellDialogue.exists).ok();
+});
+
+test('should update existing cell,should create lesson+location on empty cell', async (t) => {
+  await classesPage.createNewScheduleCell();
+  await t.click(classesPage.editCellUpdateButton);
+  //TODO - this currently fails because of bug #264
```

Comment:
Adding this teast now means that when we will activate e2e eventually master build will fail. Please comment or better remove these tests and we will add them back once e2e tests will work

Reply:
Issue #264 has been fixed and merged.